tests/servers/rendering/test_shader_preprocessor.h

```diff
@@ -52,8 +52,8 @@ bool is_variable_char(unsigned char c) {
 }
 
 bool is_operator_char(unsigned char c) {
-	static const std::string operators = "<>=+-*/";
-	return operators.find(c) != std::string::npos;
+	static const Vector<unsigned char> operators({ '<', '>', '=', '+', '-', '*', '/' });
+	return operators.find(c) != -1;
```

Comment:
It's only a few options, so does not need a vector of function calls at all, probably should be implemented similarly to the methods in the https://github.com/godotengine/godot/blob/master/core/string/char_utils.h.

Reply:
Very true will change soon, thank you!